apiary.apib

```diff
@@ -1699,6 +1699,7 @@ Get a list of all phases a deal can go through, sorted by their order in the flo
     + Attributes (object)
         + filter (object, optional)
             + ids: `53a7e592-a136-4bae-ae15-517befd3d75d`,`bb50af79-55cd-4be0-8306-e9626c70a90f` (array[string], optional)
+            + exclude_lost: true (boolean)
```

Comment:
```suggestion
            + exclude_lost: false (boolean)
```

Default value will be `false`.

Reply:
I'm seeing a double negation here. Isn't that something we try to avoid?

Reply:
@WouterSioen I indeed remember discussing the double negation part and agree with the comment... but we might need revert this commit, ask @PieterReel ...